dotCMS/src/main/java/com/dotmarketing/db/listeners/RollbackListener.java

```diff
@@ -0,0 +1,49 @@
+package com.dotmarketing.db.listeners;
+
+public class RollbackListener implements DotListener {
+    private final String key;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field key has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)